terminal/main.cpp

```diff
@@ -25,55 +29,63 @@ int main(void)
 				j++;
 				continue;
 			}
+			if (input[i] == '/')
+			{
+				path.push_back(curr);
+				curr = "";
+				continue;
+			}
+
 			if (j == 0)
 			{
 				command += input[i];
 			}
 			else if (j == 1)
 			{
-				second += input[i];
+				curr += input[i];
 			}
 			else if (j == 2)
 			{
 				third += input[i];
 			}
-
-
 		}
+		path.push_back(curr);
 
-
-		if (0 == command.compare("mkdir"))
-		{
-			t.add(second);
-		}
-		else if(0 == command.compare("exit"))
-		{
-			exit = true;
-		}
-		else if(0 == command.compare("ls"))
-		{
-			t.list(t.head);
-			
-		}
-		else if (0 == command.compare("cd") && 0 != second.compare(".."))
-		{
-			t.cd(t.head, second);
-		}
-		else if (0 == command.compare("cd") && 0 == second.compare(".."))
-		{
-			t.up(t.head);
-		}
-		else if (0 == command.compare("rm") && 0 != second.compare("-rf") && third.length() == 0)
-		{
-			t.rm(t.head, second);
-		}
-		else if (0 == command.compare("rm") && 0 == second.compare("-rf"))
-		{
-			t.rmrf(t.head, third);
-		}
-		else
+		for (string second : path)
 		{
-			cout << "Command not exist!" << endl;
+			if (0 == command.compare("mkdir"))
+			{
+				t.add(second);
```

Comment:
"long path" does not work: `mkdir alpha/beta/gamma` creates `alpha`, `beta`, and `gamma` directories in current directory  instead of `alpha/beta/gamma` path.